src/org/etools/j1939_84/bus/Packet.java

```diff
@@ -314,17 +314,14 @@ public long get64() {
     /**
      * Returns the destination address
      *
-     * @param requestPacket
-     *            the request
      * @return the destination specific address or GLOBAL_ADDR
      */
     public int getDestination() {
         return getId(0xFFFF) < 0xF000 ? getId(0xFF) : J1939.GLOBAL_ADDR;
     }
 
-    @Deprecated
     public int getId() {
-        return id;
+        return getId(0xFFFF) < 0xF000 ? getId(0xFF00) : getId(0xFFFF);
```

Comment:
This changes getId to be the same as getPgn, right?  I deprecated getId, because it was confusingly not working like getPgn.

Reply:
ok, I'll reverse this and start using getPgn()